### DISCUSSION TITLE
Vertical text for Chinese characters

### DISCUSSION THREAD

**Discussion by spoiltcheese** [QuestPDF/QuestPDF#933] [Q&A] [answered]
I have a project that requires the output of Chinese text in a vertical format, and I was wondering if QuestPDF is capable of this. I tried the ContentRightToLeft property but all I got was right aligned text. I tried searching for this topic as well but did not get any answers.

**Comment by MarcinZiabek** [comment, marked as answer]
Thank you for sharing the feedback 😄 

I am sorry to say that the top-to-bottom text direction is not currently suppored. At this moment, it is not on the upcoming roadmap.

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
